python/py-catalogue/Portfile

```diff
@@ -7,9 +7,8 @@ PortGroup           python 1.0
 epoch               1
 # Don't upgrade past py-spaCy's highest supported version.
 github.setup        explosion catalogue 2.0.7 v
-# Change github.tarball_from to 'releases' or 'archive' next update
-github.tarball_from tarball
 revision            0
+github.tarball_from releases
```

Comment:
this should only be done when updating the version. Doing so now will result in `checksum` differences unless you follow the [StealthUpdate procedure](https://trac.macports.org/wiki/PortfileRecipes#stealth-updates).

This comment applies to other commits as well!